bokehjs/test/models/axes/axis.coffee

```diff
@@ -16,10 +16,11 @@ Toolbar = utils.require("models/tools/toolbar").Model
 describe "Axis.Model", ->
 
   it "should have a SidePanel after add_panel is called", ->
-    p = new PlotCanvas({
+    pp = new Plot({
```

Comment:
This seems like an odd re-working of this test.

I think it highlights that the separation isn't quite right between Plot and PlotCanvas when it comes to the side panels. The ownership is now spread over plot and PlotCanvas. Seems to me like the logical extension of this move is that SidePanels are on Plot.